cmd/agent/subcommands/diagnose/command.go

```diff
@@ -100,8 +104,25 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 					InitHelper: common.GetWorkloadmetaInit(),
 				}),
 				fx.Supply(optional.NewNoneOption[collector.Component]()),
-				taggerimpl.Module(),
-				fx.Provide(func(config config.Component) tagger.Params { return tagger.NewTaggerParamsForCoreAgent(config) }),
+				dualTaggerfx.Module(tagger.DualParams{
```

Comment:
Now this piece of code is duplicated in several places whereas before we had a function that we could reuse. Can we fix this?

Reply:
Same comment applies to the other similar functions that we had to get the tagger params.

Reply:
I'm not opposed to that change, but I prefer to wait for helper function on a separate PR once we have merged this PR, it is already quite big.

Reply:
But in this case the helpers were already there and this PR removed them and duplicates code instead, so it seems a step backwards.